hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -3001,6 +3010,43 @@
       stage: V2
       tests: []
     }
+    {
+      name: chip_lc_scrap
+      desc: '''Ensure it is possible to enter scrap state from every legal life cycle state.
+
+            -  Request transition to SCRAP state using the JTAG interface.
+            -  It should be possible to transition from every legal state using external clock.
+            -  Where it is allowed, transition using internal clocks should also be checked.
+            -  After transition, verify that the device is in SCRAP state through LC read.
+            -  Verify while in SCRAP state:
+               - RV JTAG interface is unavailable.
+               - Ibex is not executing.
+               - RV_DM is unreachable by the stub CPU.
+
+            - X-ref'd with manuf_scrap from the manufacturing testplan.
+            - X-ref'd with chip_lc_test_locked.
+            - X-ref'd with chip_tap_strap_sampling
+            '''
+      stage: V2
+      tests: []
+    }
+    {
+      name: chip_lc_test_locked
+      desc: '''Transition from TEST_UNLOCKED to TEST_LOCKED using LC JTAG interface.
+
+            -  Check in TEST_UNLOCKED RV JTAG interface is available.
+            -  Verify When in TEST_LOCKED state:
+               - RV JTAG interface is unavailable.
+               - Ibex is not executing.
+               - RV_DM is unreachable by the stub CPU.
+
+            - X-ref'd with manuf_cp_test_lock from the manufacturing testplan.
+            - X-ref'd with chip_lc_test_locked.
```

Comment:
Is this a typo? Do you mean to link to some other test? maybe `chip_lc_scrap`?

Reply:
o yes i think you're right, let me fix that.